contrib/seccomp/seccomp_default.go

```diff
@@ -184,6 +185,9 @@ func DefaultProfile(sp *specs.Spec) *specs.LinuxSeccomp {
 				"io_uring_setup",
 				"ipc",
 				"kill",
+				"landlock_create_ruleset",
+				"landlock_add_rule",
+				"landlock_restrict_self",
```

Comment:
This still allows https://github.com/opencontainers/runtime-spec/pull/1111 protections for all containers later on, right? I think it is ok but double checking.

Reply:
Yes so, AFAIK

> If the landlock_restrict_self system call succeeds, the current thread is now restricted and this policy will be enforced on all its subsequently created children as well. Once a thread is landlocked, there is no way to remove its security policy; only adding more restrictions is allowed. These threads are now in a new Landlock domain, merge of their parent one (if any) with the new ruleset.

https://www.kernel.org/doc/html/v5.16/userspace-api/landlock.html

Reply:
Tiny nit: alphabetical order?

Reply:
done